controllers/Profile.js

```diff
@@ -16,6 +17,7 @@ exports.updateProfile = async (req, res) => {
 		profile.dateOfBirth = dateOfBirth;
 		profile.about = about;
 		profile.contactNumber = contactNumber;
+    profile.gender = gender;
```

Comment:
why are you not streamlining code formatting

Reply:
i forgot to setup in backend